src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -6,6 +6,7 @@ set DOTNET_CMD=%~2
 set TOOLRUNTIME_DIR=%~3
 IF [%BUILDTOOLS_TARGET_RUNTIME%]==[] set BUILDTOOLS_TARGET_RUNTIME=win7-x64
 set BUILDTOOLS_PACKAGE_DIR=%~dp0
+set MSBUILD_CONTENT_JSON={"dependencies": {"Microsoft.Portable.Targets": "0.1.1-dev"},"frameworks": {"dnxcore50": {},"net46": {}}}
```

Comment:
The reason is that the contents of this package needs to be copied (along with the package's structure) into the ToolRuntime/Extensions folder. If we put this in the tool-runtime project.json instead, then the contents of the package will be flattened out and dropped on the root of the ToolRuntime folder which will not work for MSBuild. I'll add a comment inline the scripts to explain this and also to mention that once dotnet cli adds support for content files then we can remove this workaround.